draft-kazuho-httpbis-priority.md

```diff
@@ -549,6 +555,17 @@ reflecting the urgencies expressed in the requests, so that less urgent
 responses would receive less bandwidth in case the bottleneck exists between the
 server and the intermediary.
 
+### HTTP/1 origins {#http11}
```

Comment:
HTTP/1 and then (#http11) ?? maybe just call it HTTP/1.x

Reply:
Unintentional error. Will resolve.